tests/rt.conf

```diff
@@ -375,34 +381,38 @@ RUN | hrrr_control_decomp_dyn32_phy32                   | + hera cheyenne
 RUN | rap_restart_dyn32_phy32                           | + hera cheyenne            |          | rap_control_dyn32_phy32
 RUN | hrrr_control_restart_dyn32_phy32                  | + hera cheyenne            |          | hrrr_control_dyn32_phy32
 RUN | hrrr_control_restart_qr_dyn32_phy32               | + hera cheyenne            |          | hrrr_control_qr_dyn32_phy32
+RUN | rrfs_smoke_conus13km_phy32                        |                            | baseline |
+RUN | rrfs_smoke_conus13km_phy32_qr                     |                            | baseline |
+RUN | rrfs_smoke_conus13km_phy32_restart_mismatch       |                            | baseline | rrfs_smoke_conus13km_phy32
+RUN | rrfs_smoke_conus13km_phy32_restart_qr_mismatch    |                            |          | rrfs_smoke_conus13km_phy32_qr
 
-COMPILE | 43 | gnu | -DAPP=ATM -DCCPP_32BIT=ON | + hera cheyenne | fv3 |
+COMPILE | atm_dyn64_phy32 | gnu | -DAPP=ATM -DCCPP_32BIT=ON | + hera cheyenne | fv3 |
 RUN | rap_control_dyn64_phy32                           | + hera cheyenne            | baseline |
 
-COMPILE | 44 | gnu | -DAPP=ATM -D32BIT=ON -DCCPP_32BIT=ON -DDEBUG=ON | + hera cheyenne | fv3 |
+COMPILE | atm_dyn32_phy32_debug | gnu | -DAPP=ATM -D32BIT=ON -DCCPP_32BIT=ON -DDEBUG=ON | + hera cheyenne | fv3 |
 RUN | rap_control_debug_dyn32_phy32                     | + hera cheyenne            | baseline |
 RUN | hrrr_control_debug_dyn32_phy32                    | + hera cheyenne            | baseline |
 
-COMPILE | 45 | gnu | -DAPP=ATM -DCCPP_32BIT=ON -DDEBUG=ON | + hera cheyenne | fv3 |
+COMPILE | atm_dyn64_phy32_debug | gnu | -DAPP=ATM -DCCPP_32BIT=ON -DDEBUG=ON | + hera cheyenne | fv3 |
 RUN | rap_control_dyn64_phy32_debug                     | + hera cheyenne            | baseline |
 
 ### S2S tests ###
-COMPILE | 46 | gnu | -DAPP=S2SWA -DCCPP_SUITES=FV3_GFS_v17_coupled_p8 | + hera cheyenne | fv3 |
+COMPILE | s2swa | gnu | -DAPP=S2SWA -DCCPP_SUITES=FV3_GFS_v17_coupled_p8 | + hera cheyenne | fv3 |
```

Comment:
There is already a swswa in the "Intel" section, I think this will try to use compile_swswa directory and if there is already one, will cause complications if they are running at the same time. If there are similar compile names maybe add _gnu to the GNU compiles?

Reply:
It already adds _gnu or _intel to the end of the compile_nr. If you type `s2swa`, you get `s2swa_gnu` or `s2swa_intel`.

Reply:
Also, it checks for duplicate compilation names. It uses hashes, which require bash 4.2 or later. Older bash versions will abort due to syntax errors.

Reply:
OK, sounds good to me. Thanks for the clarification.